lib/cisco_node_utils/cmd_ref/itd_device_group.yaml

```diff
@@ -0,0 +1,83 @@
+# itd_device_group
+---
+_exclude: [N3k, N5k, N6k, ios_xr]
+
+_template:
+  get_command: "show running-config all | section itd"
+  get_context:
+      - '/^itd device-group <name>/'
+      - '(?)/^node <ntype> <nname>/'
```

Comment:
It is not just yaml but the ruby code as well. Currently with this design, the ruby code also does not duplicate. When standby ip provider comes for itd (it is buggy right now so we deferred), the same code gets duplicated yet again so we will have 3 copies of the same thing (yaml file and ruby) if we break up files.